runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -141,6 +141,18 @@
     private String usernameFile;
     private String passwordFile;
 
+
+    /**
+     * Trigger the loading of all the SmrMaps upon connect.
+     *
+     * If using this utility, you need to be sure that no one
+     * is acessing objects until the tables are loaded
+     * (i.e. when connect return)
```

Comment:
I think this should be more runtime level fencing and not across runtimes. This event is contained within a runtime.